src/android/NetworkManager.java

```diff
@@ -96,15 +115,17 @@ public class NetworkManager extends CordovaPlugin {
      * @param webView The CordovaWebView Cordova is running in.
      */
     public void initialize(CordovaInterface cordova, CordovaWebView webView) {
-        super.initialize(cordova, webView);
-        this.sockMan = (ConnectivityManager) cordova.getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
-        this.connectionCallbackContext = null;
+    super.initialize(cordova, webView);
+    this.sockMan = (ConnectivityManager) cordova.getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
+    this.telMan = (TelephonyManager) cordova.getActivity().getSystemService(Context.TELEPHONY_SERVICE);
+    this.telMan.listen( phoneStateListener, LISTEN_SERVICE_STATE);
+    this.connectionCallbackContext = null;
```

Comment:
```suggestion
        super.initialize(cordova, webView);
        this.sockMan = (ConnectivityManager) cordova.getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
        this.telMan = (TelephonyManager) cordova.getActivity().getSystemService(Context.TELEPHONY_SERVICE);
        this.telMan.listen( phoneStateListener, LISTEN_SERVICE_STATE);
        this.connectionCallbackContext = null;
```

Same here. It seems to not be indented at all within the method block.